letsencrypt/auth_handler.py

```diff
@@ -113,6 +136,14 @@ def _solve_challenges(self):
                     cont_resp = self.cont_auth.perform(self.cont_c)
                 if self.dv_c:
                     dv_resp = self.dv_auth.perform(self.dv_c)
+                    for achall, response in zip(self.dv_c, dv_resp):
+                        if not self._simple_verify(achall, response):
+                            logger.warning("Self-verify of %s challenge (%s) "
+                                           "for domain %s failed.",
+                                           achall.chall.__class__.__name__,
+                                           achall.domain,
+                                           achall.uri)
```

Comment:
Please make the line wrapping optimally compact by:
- putting more arguments on the same line, e.g. `achall.domain, achall.uri` instead of `achall.domain,\nachall.uri`
- you can try to put a newline after `logger.warning(` and then continue with 4 space indent.